.github/workflows/ci.yml

```diff
@@ -65,32 +55,21 @@ jobs:
 
     - name: Install Rust wasm32-wasi
       uses: actions-rs/toolchain@v1.0.7
+
       with:
         toolchain: stable
         target: wasm32-wasi
         override: true
 
     - name: Install a recent version of clang
       run: |
-        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
-        echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main" >> /etc/apt/sources.list
         apt-get update
         apt-get install -y clang-11
 
     - name: Install CMake
       run: apt-get install -y cmake
 
-    - name: Cache CARGO_HOME
-      uses: actions/cache@v2.1.6
-      with:
-        path: ~/.cargo
-        key: cargo-home-${{ hashFiles('Cargo.toml') }}
-
-    - name: Cache cargo build
-      uses: actions/cache@v2.1.6
-      with:
-        path: target
-        key: wasm-cargo-build-target-${{ hashFiles('Cargo.toml') }}
+    - uses: Swatinem/rust-cache@v1
```

Comment:
Same as with the `checkout` action, specifying the full version will avoid a direct dependabot PR after this is merged.